client/src/components/MailBox.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import axios from 'axios'
+import * as Yup from 'yup'
+import { Formik, Form } from 'formik';
+import FormField from "../constant/FormField";
+import StyledButton from '../constant/StyledButton';
+import styled from 'styled-components'
+
+const MailBox = ({receivers, setMBox, setMailSuccess}) => {
+    console.log('mail box receiver', receivers)
+
+    const handleSubmit = async (values)=>{
+        console.log('mail values', values)
+        const config = {
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		};
+		await axios.post('https://designed-gd.herokuapp.com/api/send', values, config)
+		setMBox(false)
+		setMailSuccess(true)
+    }
+
+    const initialValue ={sender:'cyf.graduate.platform@gmail.com', receiver:receivers, subject:'', message:''}
+
+    return (
+        <Container>
+            <Formik
+				initialValues={initialValue}
+				onSubmit={(values) => handleSubmit(values)}
```

Comment:
You should be able to shorten this:
```suggestion
				onSubmit={handleSubmit}
```